app/src/node.rs

```diff
@@ -218,6 +218,7 @@ impl Node for App {
             max_retain_blocks,
             prune_at_block_interval,
             min_block_time,
+            emerald_config.fee_recipient,
```

Comment:
Why do we put this in the state if it's part of the config? Do we have logic to change the value in state? If it's part of the config, it seems to me it should change when the node restarts with a new config, right?